disk/tests/test_unit.py

```diff
@@ -155,3 +156,29 @@ def test_min_disk_size(aggregator, gauge_metrics, rate_metrics):
         aggregator.assert_metric_has_tag(name, 'device:{}'.format(DEFAULT_DEVICE_NAME))
 
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')
```

Comment:
I think `@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')` is not needed for those 2 tests since we are reading files.